### DISCUSSION TITLE
Error Running in Windows Git Bash

### DISCUSSION THREAD

**Discussion by PredatoryEyesLOL** [pystardust/ani-cli#456] [Q&A] [answered]
It says there is a command jq missing in line 92 sometimes and sometimes it opens the vlc but the vlc shows only errors and it was running a dll file named libcarro2 or something and also some exe file of a program I have installed in my Laptop. Here is a snip of jq error to make it clear. ![image](https://user-images.githubusercontent.com/84143869/151702092-c61a9a79-5f8a-415e-a395-377e9c9bef7c.png)

**Comment by port19x** [comment, marked as answer]
During the time of your post, `jq` was an external dependency you would have to install.
Now we even offer a scoop package, so it should be a lot easier to install now.

**Reply by PredatoryEyesLOL** [reply]
Thanks I did run new installation by using scoop package and it runs without any hiccups.
